eng/scripts/Test-Packages.ps1

```diff
@@ -52,7 +52,7 @@ foreach ($package in $packagesToTest) {
     Write-Host "Checking for setup in $serviceDirectory + 'Test-Setup.ps1'"
     if (Test-Path ($serviceDirectory + "Test-Setup.ps1")) {
       Write-Host "`n`nRunning test setup script for package: '$($package.Name)': ($serviceDirectory)Test-Setup.ps1`n"
-      Invoke-LoggedCommand "($serviceDirectory)Test-Setup.ps1 -packageName $package.Name -workingDirectory $WorkingDirectory"
+      Invoke-LoggedCommand "($serviceDirectory)Test-Setup.ps1 -packageName ($package.Name) -workingDirectory $WorkingDirectory"
```

Comment:
That won't work either. In a string, you need to use `$(...)` to execute an expression.

Reply:
Yeah, that's been a struggle... But I finally got something that works. Probably not the most straightforward but it works.